app/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -159,7 +163,42 @@ const TableAudience = ({
               Recipe JSON <a href={`#recipe-json`}>#</a>
             </th>
             <td data-testid="experiment-recipe-json">
-              <Code codeString={experiment.recipeJson} />
+              <Accordion>
+                <Accordion.Toggle
+                  as={Accordion}
+                  eventKey="0"
+                  onClick={() => setExpand(!expand)}
+                >
+                  {expand ? (
+                    <>
+                      <div className="float-right">
+                        <CollapseMinus />
+                        <span style={{ textTransform: "capitalize" }}>
+                          Hide
+                        </span>
```

Comment:
Oh thank you yes putting it over there does make it much nicer to use, but I just realized, using a `span` here feels kind of awkward since it's a clickable element but uses the text cursor instead of the click cursor.  Maybe like a `button` or `label` or something is more appropriate?

The [examples](https://react-bootstrap.github.io/components/accordion/#custom-toggle-with-expansion-awareness) use a button for instance, can you see if you can find something that's more clickable? 🙏